test/Microsoft.AspNetCore.SignalR.Tests/ServerFixture.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.PlatformAbstractions;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.SignalR.Tests
+{
+    public class ServerFixture : IDisposable
+    {
+        private readonly bool _verbose;
+
+        private Lazy<string> _baseUrl;
+        private object _lock = new object();
+        private ILoggerFactory _loggerFactory;
+        private IApplicationDeployer _deployer;
+
+        public string BaseUrl => _baseUrl.Value;
+
+        public string ServerProjectName
+        {
+            get { return "Microsoft.AspNetCore.SignalR.Test.Server"; }
+        }
+
+        public ServerFixture()
+        {
+            _loggerFactory = new LoggerFactory();
+
+            _verbose = string.Equals(Environment.GetEnvironmentVariable("SIGNALR_TESTS_VERBOSE"), "1");
```

Comment:
This is the only use of `_verbose` probably don't need to make it a class member.